modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -152,6 +156,38 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		FolderArticleModifiedDateComparator.getInstance(false);
 		FolderArticleTitleComparator.getInstance(false);
 		FolderIdComparator.getInstance(false);
+		FolderIdComparator.getInstance(false);
```

Comment:
There is already a test case for `FolderIdComparator`

Reply:
Thanks @Valmir-unicap! Another class with the same name was merged, so the `FolderIdComparator` became duplicated. I fixed it.